lms/djangoapps/certificates/generation_handler.py

```diff
@@ -93,7 +93,7 @@ def generate_allowlist_certificate_task(user, course_key, generation_mode=None):
     Create a task to generate an allowlist certificate for this user in this course run.
     """
     if _can_generate_allowlist_certificate(user, course_key):
-        return _generate_certificate_task(user, course_key, generation_mode)
+        return _generate_certificate_task(user=user, course_key=course_key, generation_mode=generation_mode)
```

Comment:
Sorry that I missed this originally. I'll have to take a closer look at keyword arguments in functions in the future.

Nit: I don't think we need the `user=` and `course_key=` declarations in the function calls as they are positional arguments in the function unless you _want_ it to be explicit. I don't feel strongly either way.